packages/@tinacms/app/src/index.ts

```diff
@@ -150,6 +150,7 @@ export const viteBuild = async ({
       target: 'es2020',
     },
     server: {
+      host: true,
```

Comment:
Is this a good default to have? Would it be better allow either an env variable or CLI flag to set this? And otherwise leave it off?

Reply:
That makes sense.